pickapp/src/main/kotlin/com/lovoo/android/pickapp/view/PickPicActivity.kt

```diff
@@ -57,6 +55,7 @@ class PickPicActivity : AppCompatActivity(), SelectionHolder, CameraEngine, Capt
 
     private lateinit var config: PickPicConfig
     private lateinit var picker: Picker
+    private lateinit var binding: PickpicActivityBinding
```

Comment:
ModulesBaseActivity has AppCompatActivity as parent, cant you just change parent class here and use ModulesBaseActivity<PickpicActivityBinding> to have coherent solutions?

Reply:
This is not lovoo, there is no ModulesBaseActivity